source/common/ratelimit/ratelimit.proto

```diff
@@ -2,8 +2,6 @@ syntax = "proto3";
 
 option go_package = "ratelimit";
 
-option cc_generic_services = true;
```

Comment:
Yeah, I think we want to cut to the v2 API when we switch to envoy-api, and there we have a different package namespace etc., so that's best left as future work.